<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OivObjectenDockWidgetBase</class>
 <widget class="QDockWidget" name="OivObjectenDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>576</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
  </property>
  <property name="windowTitle">
   <string>OIV Objecten</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>316</width>
         <height>534</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <weight>75</weight>
              <bold>true</bold>
              <underline>true</underline>
             </font>
            </property>
            <property name="text">
             <string>U heeft het volgende repressief object geselecteerd:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="formelenaam">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(208, 208, 208);
border-color: rgba(255, 255, 255, 0);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="object_id">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(208, 208, 208);
border-color: rgba(255, 255, 255, 0);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="objectgegevens">
            <property name="text">
             <string>Objectgegevens bewerken</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="delete_object">
            <property name="text">
             <string>Object verwijderen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="terugmelden">
            <property name="text">
             <string>Terugmelden BGT en/of BAG</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="object_symbolen">
            <property name="text">
             <string>Object tekenen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="create_grid">
            <property name="text">
             <string>Maak een Kaartblad en/of Grid aan</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="import_drawing">
            <property name="text">
             <string>Importeer externe tekening</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="terrein_bewerken">
            <property name="text">
             <string>Object terrein bewerken</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <item row="1" column="0" colspan="3">
             <widget class="QLabel" name="straal_label">
              <property name="text">
               <string>Hulpcirkel (m)</string>
              </property>
             </widget>
            </item>
            <item row="2" column="4">
             <widget class="QDoubleSpinBox" name="offset">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="minimum">
               <double>-1000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="straal_button">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/plugins/oiv/config_files/png/hulpcirkel.png</normaloff>:/plugins/oiv/config_files/png/hulpcirkel.png</iconset>
              </property>
             </widget>
            </item>
            <item row="0" column="3" colspan="2">
             <widget class="QLabel" name="oppervlakte_label">
              <property name="text">
               <string>Oppervlakte (m2)</string>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QDoubleSpinBox" name="oppervlakte">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(240, 240, 240);
border-color: rgba(255, 255, 255, 0);</string>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="maximum">
               <double>100000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QPushButton" name="offset_button">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/plugins/oiv/config_files/png/offset_parallel.png</normaloff>:/plugins/oiv/config_files/png/offset_parallel.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="lengte_label">
              <property name="text">
               <string>Lengte (m)</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="terrein_tekenen">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>open formulier van een object...</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/plugins/oiv/config_files/png/terrein.png</normaloff>:/plugins/oiv/config_files/png/terrein.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="default">
               <bool>false</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="delete_f">
              <property name="toolTip">
               <string>delete object...</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/plugins/oiv/config_files/png/delete.png</normaloff>:/plugins/oiv/config_files/png/delete.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QPushButton" name="identify">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>open formulier van een object...</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/plugins/oiv/config_files/png/identify.png</normaloff>:/plugins/oiv/config_files/png/identify.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="default">
               <bool>false</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="straal">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="maximum">
               <double>1000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="4">
             <widget class="QPushButton" name="pan">
              <property name="toolTip">
               <string>selecteer de functie Pan...</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/plugins/oiv/config_files/png/pan.png</normaloff>:/plugins/oiv/config_files/png/pan.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QDoubleSpinBox" name="lengte">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(240, 240, 240);
border-color: rgba(255, 255, 255, 0);</string>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="maximum">
               <double>100000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="3" colspan="2">
             <widget class="QLabel" name="offset_label">
              <property name="text">
               <string>Offset parallel (m)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="terug">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Gereed</string>
            </property>
            <property name="iconSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
